dev/com.ibm.ws.security.openidconnect.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -298,3 +298,8 @@ pkceCodeChallengeMethod.S256=S256
 
 tokenRequestOriginHeader=Token request origin header
 tokenRequestOriginHeader.desc=Specifies the value to use in the Origin HTTP header that is included in the HTTP POST request to the token endpoint of the OpenID Connect provider. If not specified, an Origin HTTP header is not included in the request.
+
+tokenOrderToFetchCallerClaims=Specify one or more tokens to fetch caller claims 
+tokenOrderToFetchCallerClaims.desc=Specifies the order of the token/s to fetch the caller name and group claim values. If the claim does not exist in the first token, then the OpenID Connect client will continue the search with other tokens in the list
+tokenOrderToFetchCallerClaims.IDToken=Use IDToken only to determine the caller claims
+tokenOrderToFetchCallerClaims.ACCESSAndIDAndUITokens=Use AccessToken, IDToken and Userinfo tokens in this order, to determine the caller claims.
```

Comment:
Add a comma after "IDToken" and remove the comma after "order" >>

"Use AccessToken, IDToken, and Userinfo tokens in this order to determine the caller claims."

Reply:
addressed this review comment in the runtime , config PR - #26719

Reply:
#26719 looks good. I'll add the ID reviewed label to this PR. Thanks for making the changes!